src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -254,7 +242,8 @@ public void AddDiscoveredNodesRequirement(NetworkPeerServices services)
         {
             this.logger.LogTrace("({0}:{1})", nameof(services), services);
 
-            this.discoveredNodeRequiredService |= services;
+            var discoveredNodeRequiredService = NetworkPeerServices.Network;
```

Comment:
`NetworkPeerServices.Network` means the node can serve blocks
This is probably very wrong, you can't know at this point if the node service is Network, only the BlockStore can do that as only when block store is present the node can serve blocks to peers

Reply:
Whats the reason behind setting this here and not in store? did you hit an issue?

Reply:
`discoveredNodeRequiredService` was actually set to `NetworkPeerServices.Network` at field declaration. See a bit higher up in the diff.... But I'm going to revert this bit of code rather 👍